Lesson16/selcan/script.js

```diff
@@ -0,0 +1,304 @@
+// 9. Write a function that takes in a person's age and checks if they are a child (age 0-12), teenager (age 13-19), adult (age 20-64), or senior citizen (age 65 and above) using if else statement.
+function isChild(age){
+	if (age>0 && age<12){
+		return "Exercise9:Child";
+	} else if(age>13 && age<19){
+	    return "Exercise9:Teenage";
+    } else if(age>20 && age<64){
+	    return "Exercise9:Adult";
+    } else if(age>65){
+	    return "Exercise9:Senior citizen";}
+	}
+
+	console.log(isChild(34));
+
+// 10. Write a function that takes age as a number and whether they are a student (yes or no). If the age is less than 18 and the user is a student, print "You are a student and not old enough to vote". If the age is between 18 and 65 (inclusive) and the user is not a student, print "You are eligible to vote". If the age is greater than 65, print "You are a senior citizen". If the user is a student but their age is greater than or equal to 18, print "You are old enough to vote".
+
+let kisi = "student";
+function canVote(age){
+	if (age<18 && kisi){
+		return "You are a student and not old enough to vote";}
+	else if (age>18 && age<65 && !kisi){
+		return "You are eligible to vote";}
+    else if(age>65){
+	    return "You are a senior citizen";}
+	else if(age>=18 && kisi){
+		return"You are old enough to vote";}
+	}
+	console.log(canVote(34));
+	
+// 11. Write a function that asks the user to enter their salary and their years of experience. If the salary is less than $30,000 and the years of experience are less than 5, print "Sorry, we cannot offer you the job at this time". If the salary is less than $50,000 and the years of experience are between 5 and 10 (inclusive), print "We can offer you the job at a lower salary". If the salary is greater than or equal to $50,000 or the years of experience are greater than or equal to 10, print "We can offer you the job at the requested salary".
+function offerJob(salary,experience){
+	if (salary<30.000 && experience<5){
+		return "Sorry, we cannot offer you the job at this time";
+	} else if(salary<50.000 && experience>5 && experience<10){
+		return "We can offer you the job at a lower salary";
+	} else if(salary>=50.000 && experience>=10){
+		return "We can offer you the job at the requested salary";
+	}
+}
+console.log(offerJob(salary,experience));
+
+
+// 12. Write a function that asks the user to enter their score on a test. If the score is less than 60, print "You failed the test". If the score is between 60 and 70 (inclusive), print "You passed the test, but your grade is not great". If the score is between 71 and 80 (inclusive), print "You got a B". If the score is between 81 and 90 (inclusive), print "You got an A". If the score is greater than 90, print "You got an A+ - Great job!".
+function score(number){
+	if (number<60){
+		return "You failed the test";
+	} else if(number>=60 && number<=70){
+		return "You passed the test, but your grade is not great";
+	} else if(number>71 && number<=80){
```

Comment:
71 should be included otherwise it will be skipped in this function
```suggestion
	} else if(number>=71 && number<=80){
```